Kickstarter-iOS/Library/SharedFunctionsTests.swift

```diff
@@ -481,7 +482,8 @@ internal final class SharedFunctionsTests: TestCase {
     XCTAssertEqual(max, 200_000)
   }
 
-  func testMinAndMaxPledgeAmount_Reward_ProjectCurrencyCountry_MinMaxPledgeReturned_Success() {
+  func testMinAndMaxPledgeAmount_Reward_ProjectCurrencyCountry_isNotLatePledge_MinimumPledgeReturned_Success(
```

Comment:
Can you please add a test similar to this one (where there's no backing but with reward.pledgeAmount and reward.latePledgeAmount set) where the project is in a late pledge state? See comment in the shared functions file itself for my concern that this case might not work as intended.